src/FirebirdSql.Data.FirebirdClient/Types/FbZonedDateTime.cs

```diff
@@ -72,8 +72,50 @@ public override int GetHashCode()
 		}
 	}
 
-	public bool Equals(FbZonedDateTime other) => DateTime.Equals(other.DateTime) && TimeZone.Equals(other.TimeZone, StringComparison.OrdinalIgnoreCase);
-
+	public bool Equals(FbZonedDateTime other) => DateTime.Equals(other.DateTime) && TimeZone.Equals(other.TimeZone, StringComparison.OrdinalIgnoreCase);

```

Comment:
Why is this showing as changed? 🤷‍♂️

Reply:
Have you seen this before? I don't like the idea of ignoring it, and would like to know if I did something wrong here.

Reply:
This does not show as changed when I look at the diff locally.